introduction/java_static_initializer_block/Solution.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Scanner;
+
+public class Solution {
+  private static boolean flag = true; 
+  public static int breadth;
+  public static int height;
+  static { 
```

Comment:
NL is used to separate logical blocks. In this case the first logical block is declaration of fields, the second block - static block and the third block - main function. So they should be separated from each other with NLs.